Automattic/wp-calypso
# ETK: wpcom-block-description-links - Update button text

Reference discussion: https://github.com/Automattic/wp-calypso/pull/86067#issuecomment-1880460913

## Proposed Changes

* Update button text from "Learn more" to "Block guide"

## Testing Instructions



* Checkout branch
* Navigate to apps/editing-toolkit and run yarn dev --sync
* Sandbox your site and public api.
* Navigate to post editor , add block, select the block, open the settings menu and verify the button text is "Block guide" (screenshot below) 

<img width="1149" alt="CleanShot 2024-01-11 at 19 58 09@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/42061fda-bb57-4d14-ad89-f5f68762d1aa">


## Pre-merge Checklist
